.github/workflows/tests.yml

```diff
@@ -119,24 +129,63 @@ jobs:
 
       - name: Start Covalent dispatcher server
         run: covalent start
+               
+      - name: Run SDK tests and measure coverage
+        if: ${{ steps.working-directory.outputs.sdk}} == 'true'
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 10
+          max_attempts: 2
+          on_retry_command: rm -f ~/.config/covalent/covalent.conf
+          command: |
+            PYTHONPATH=$PWD/ pytest -vv tests/covalent_tests  --cov=covalent --cov-config=.coveragerc
```

Comment:
we cannot use multi-line commands with the retry action due to a bug.  I would suggest using the "&&" to make this a single command (although perhaps written on multiple lines).

Reply:
Thanks @wjcunningham7 @scottwn for the suggested updates